src/cargo/ops/cargo_clean.rs

```diff
@@ -104,7 +105,7 @@ pub fn clean(ws: &Workspace<'_>, opts: &CleanOptions<'_>) -> CargoResult<()> {
     // Doc tests produce no output.
 
     // Get Packages for the specified specs.
-    let mut pkg_ids = Vec::new();
+    let mut pkg_ids = BTreeSet::new();
```

Comment:
Should we instead use `IndexSet` to preserve the users order?

Reply:
An internal call to get_many() from download_accessible() in package.rs uses a BTreeSet, so that's what I used here. I don't know enough to say whether an IndexSet is warranted.

Reply:
I'd lean towards keeping ordering.

Reply:
Done.